src/main/java/com/dev/cinema/controller/MovieController.java

```diff
@@ -27,7 +28,7 @@ public MovieController(MovieService movieService, MovieMapper movieMapper) {
     }
 
     @PostMapping
-    public void create(@RequestBody MovieRequestDto movieRequestDto) {
+    public void create(@RequestBody @Valid MovieRequestDto movieRequestDto) {
```

Comment:
again, let's also validate dto in MovieSessionController

Reply:
done